app/javascript/src/theme.js

```diff
@@ -0,0 +1,1764 @@
+import StackedMenu from './stacked-menu';
+
+class Theme {
```

Comment:
Isn't this the same file as what's in our vendor and in `javascript/components`?

Reply:
No longer used